scene/resources/audio_stream_wav.cpp

```diff
@@ -624,7 +624,7 @@ Error AudioStreamWAV::save_to_wav(const String &p_path) {
 	}
 
 	String file_path = p_path;
-	if (!(file_path.substr(file_path.length() - 4, 4) == ".wav")) {
+	if (!(file_path.substr(file_path.length() - 4, 4).to_lower() == ".wav")) {
```

Comment:
```suggestion
	if (file_path.substr(file_path.length() - 4, 4).to_lower() != ".wav") {
```
Worth cleaning this up while we're here, more readable

Reply:
Great catch, thats indeed more readable :)